File: 2022/09/10158.js
const input = require("fs").readFileSync("/dev/stdin").toString().trim().split("\n");
const [W, H] = input[0].split(" ").map(Number);
let [P, Q] = input[1].split(" ").map(Number);
const T = parseInt(input[2]);
P += T;
Q += T;
let lastP = parseInt(P / W) % 2 ? W - (P % W) : P % W;
let lastQ = parseInt(Q / H) % 2 ? H - (Q % H) : Q % H;
console.log(lastP, lastQ);

File: 2022/09/10158.py
input = __import__('sys').stdin.readline
w, h = map(int, input().split())
p, q = map(int, input().split())
t = int(input())
q += t
p += t
lastP = w-(p % w) if p//w % 2 else p % w
lastQ = h-(q % h) if q//h % 2 else q % h
print(lastP, lastQ)
